public/static/js/spawned_hosts.js

```diff
@@ -363,6 +393,29 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope','$window', '$timeout', 'mciSp
           $scope.modalOpen = false;
           $scope.$apply();
         });
+      } else if ($scope.modalOption === 'stopHost') {
+        $scope.modalTitle = 'Stop Host';
+        modal.on('shown.bs.modal', function() {
+          $scope.modalOpen = true;
+          $scope.$apply();
```

Comment:
Do you specifically need to rerender the UI based on the data here? I wouldn't think so. Can you verify by removing this line and the other lines to $apply, and seeing if the escape key still dismisses the modal?

Reply:
Doesn't seem necessary! Removed them.